src/SQLServerQuery.php

```diff
@@ -43,27 +47,10 @@ public function __destruct()
         }
     }
 
-    public function getIterator()
```

Comment:
Why has this generator been replaced with the old style API? https://github.com/silverstripe/silverstripe-mssql/pull/49

Reply:
@dhensby I take it `getIterator` is for SS5 not SS4 though? I was trying to get compatibility with SS 4.6

Reply:
Hmm - ok, then we need to do some branch management on this repo, it seems.